3.7/arangosearch-views.md

```diff
@@ -294,7 +297,31 @@ During view modification the following directives apply:
   iterates over all documents of a View, wants to sort them by attribute values
   and the (left-most) fields to sort by as well as their sorting direction match
   with the *primarySort* definition, then the `SORT` operation is optimized away.
-  Also see [Primary Sort Order](arangosearch-views.html#primary-sort-order)
+  Also see [Primary Sort Order](#primary-sort-order)
+
+- **primarySortCompression** (_optional_; type: `string`; default: `lz4`; _immutable_)
+
+  Defines how to compress the primary sort data (introduced in v3.7.0).
+  ArangoDB v3.5 and v3.6 always compress the index using LZ4.
+
+  - `"lz4"` (default): use LZ4 fast compression.
+  - `"none"`: disable compression to trade space for speed.
+
+- **storedValues** (_optional_; type: `array`; default: `[]`; _immutable_)
+
+  An array of objects to describe which document attributes to store in the
+  View index. It can then cover search queries, which means the data can be
+  taken from the index directly and accessing the storage engine can be avoided.
+
+  Each object is expected in the form
+  `{ field: [ "attr1", "attr2", ... "attrN" ], compression: "none" }`,
```

Comment:
It is decided we will rename field -> fields 
https://github.com/arangodb/backlog/issues/697

Reply:
Updated